src/engraving/rendering/dev/scorehorizontalviewlayout.cpp

```diff
@@ -300,8 +300,8 @@ void ScoreHorizontalViewLayout::collectLinearSystem(LayoutContext& ctx)
                 MeasureLayout::removeSystemTrailer(m, ctx);
             }
 
-            if (m->tick() >= ctx.state().startTick() && m->tick() <= ctx.state().endTick()) {
-                // for measures in range, do full layout
+            if ((m->tick() >= ctx.state().startTick() && m->tick() <= ctx.state().endTick()) || m->mmRestCount()) {
+                // for measures in range (or multi-measure rests), do full layout
```

Comment:
Not that checking `m->mmRest()` doesn't work, it's still `nullptr` at this point.  And while this may trigger more layout than would otherwise be necessary, I wouldn't think it's expected users would commonly need to make changes to MM rests in continuous view (I don't see any performance issues with the testing I did).